### PR TITLE
Remove all gists from demos

### LINKED ISSUE #225: Code in GitHub Gists don't scale when base font size increased to 32px
Seen on Picture element but probably a general issue. Not sure if applying an override will be fragile.

### PR DESCRIPTION
**Pull request by melanierichards**
Fixes #225

### CODE REVIEW

**Comment by molant** [comment]
If I've understood correctly we are removing gift because they don't scale when we increase the size. It's that correct? Is this the only solution we can come up with? Is this a problem with the platform or with github?

**Comment by melanierichards** [comment]
We wanted to get rid of them anyway. They're inconsistent with other code samples (`code` inside a `pre`) and why bother loading scripts just for code samples?

File: webaudiotuner/index.html [hunk 1]
```diff
@@ -33,125 +33,197 @@ <h2 class="visually-hidden">Tuner</h2>
...
+var sourceAudioNode = audioContext.createOscillator();
+sourceAudioNode.frequency.value = notesArray[currentNoteIndex].frequency;
+sourceAudioNode.connect(audioContext.destination);
+sourceAudioNode.start();
+
+// This function is called passing either a +2 or a -2 to increase or decrease
+// the A4 frequency we are using as a reference
+var changeBaseFreq = function(delta) {
+	var newBaseFreq = baseFreq + delta;
+	if(newBaseFreq >= 432 && newBaseFreq <= 446) {
```
Comment (dstorey) [inline webaudiotuner/index.html]: amp Probably needs escaped here

File: webaudiotuner/index.html [hunk 2]
```diff
@@ -33,125 +33,197 @@ <h2 class="visually-hidden">Tuner</h2>
...
+			}
+	}
+};
+
+// This function is used to change the note we are currently playing, using 
+// whatever the current set of notes is (changed by the function above)
+var changeReferenceSoundNote = function(delta) {
+	if(isRefSoundPlaying) {
+			var newNoteIndex = currentNoteIndex + delta;
+			if(newNoteIndex >= 0 && newNoteIndex < notesArray.length) {
```
Comment (dstorey) [inline webaudiotuner/index.html]: same here

File: webaudiotuner/index.html [hunk 3]
```diff
@@ -164,23 +236,42 @@ <h3 class="title">Media Stream</h3>
...
-                            <p>From this API we only use one specific function to access the audio input device, generally the microphone: <strong>getUserMedia</strong>. Just like with AudioContext, we whould consider the possibility that the API may be prefixed in some browsers or older versions of them.</p>
+							<p>From this API we only use one specific function to access the audio input device, generally the microphone: <strong>getUserMedia</strong>. Just like with AudioContext, we whould consider the possibility that the API may be prefixed in some browsers or older versions of them.</p>
 						</div>
 						<div class="module module--primary">
 							<h5>Example</h5>
-                            <script src="https://gist.github.com/quimbs/cedded5dcebba82421e6.js"></script>
-                        </div>
+<pre><code>var isGetUserMediaSupported = function(){
+	navigator.getUserMedia = navigator.getUserMedia || navigator.webkitGetUserMedia || navigator.mozGetUserMedia;
+	if((navigator.mediaDevices && navigator.mediaDevices.getUserMedia) || navigator.getUserMedia){
```
Comment (dstorey) [inline webaudiotuner/index.html]: ...and again

File: webaudiotuner/index.html [hunk 4]
```diff
@@ -164,23 +236,42 @@ <h3 class="title">Media Stream</h3>
...
+			return true;
+	}
+	
+	return false;
+};
+
+if(isGetUserMediaSupported()){
+	var getUserMedia = navigator.mediaDevices && navigator.mediaDevices.getUserMedia ?
```
Comment (dstorey) [inline webaudiotuner/index.html]: ...and again

File: @supports/index.html [hunk 5]
```diff
@@ -41,20 +41,16 @@ <h3 class="title">Declaration Filters</h3>
 					<div class="subsection__body">
 						<div class="layout layout--equal">
 							<div class="module">
-								<h5>CSS</h5>
-								<script
-									src="https://gist.github.com/aarongustafson/d93fe2e887261309c03a.js?file=rgba.css"></script>
-								<noscript>
-									<pre><code class="css">b {
+								<h4>CSS</h4>
+<pre><code class="css">b {
```
Comment (dstorey) [inline @supports/index.html]: This and others in this file have class="css" but css in other files and the html and js ones don;t. are we using any library that hooks into the css class to add highlighting?

**Review by dstorey — CHANGES_REQUESTED** [review]
question about class and pointing out encoding issue for amps

File: @supports/index.html (hunk 5 above)
Comment (melanierichards) [reply @supports/index.html]: Hrm. Maybe I did weird copy-pasting from elsewhere? Not sure, removing.

**Comment by melanierichards** [comment]
@dstorey made the changes! This is ready to review again.

**Review by dstorey — APPROVED** [review]

**Pull request merged**